consensus.go

```diff
@@ -413,9 +446,13 @@ func (p *Pbft) runAcceptState(ctx context.Context) { // start new round
 // runValidateState implements the Validate state loop.
 //
 // The Validate state is rather simple - all nodes do in this state is read messages and add them to their local snapshot state
-func (p *Pbft) runValidateState(ctx context.Context) { // start new round
+func (p *Pbft) runValidateState(ctx context.Context, start time.Time) { // start new round
```

Comment:
Why is this necessary?

Reply:
ok, will move state duration updates outside the state functions and keep previous signature